tests/system/model_monitoring/test_app.py

```diff
@@ -396,6 +396,9 @@ def custom_setup(cls) -> None:
     def _submit_controller_and_deploy_writer(
         self, deploy_histogram_data_drift_app
     ) -> None:
+        self.project.set_model_monitoring_credentials(
+            endpoint_store_connection="v3io", stream_path="v3io", tsdb_connection="v3io"
+        )
```

Comment:
Note that this test should pass with other targets such as TDEngine. I guess it will be break with this change.